rbac/models.py

```
@@ -0,0 +1,1009 @@
+from django.contrib.auth.models import AbstractBaseUser, PermissionsMixin, BaseUserManager
+from django.db import models
+from django.utils import timezone
+from django.core.validators import RegexValidator
+import uuid
+from typing import Optional
+
+
+class UnifiedUserManager(BaseUserManager):
+    """مدیر کاربر یکپارچه"""
+    
+    def create_user(
```

Ruff (0.12.2)

Missing return type annotation for public function `create_user`

(ANN201)